03-transactions.md

```diff
@@ -822,292 +820,292 @@ And here are the test vectors themselves:
-    output htlc_timeout_tx 3: 0200000000010182fafbf148ac73a8a8cbbdf459408e1d368b4a2498bb0305a703cb800ae03b0e03000000000000000001f1090000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500483045022100815d4ea550897aac19da74f77e057ef0ff5640714a5a138943533111326b71650220136189c225d4981f68f53aa2815beb788ee4d072e8290cb99afa301a57f5f5d301483045022100a074832b8f1ab20a623fb4aa5e1173001f04c7b2cbcc9b87572c1c12a1bf3b1f02200a47b31262dbea329ae0f70b278c34b55176993d038f57a264e10b62b9d017aa01006921039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c820120876475527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae67a9148a486ff2e31d6158bf39e2608864d63fefd09d5b88ac68f7010000
-    # local_signature = 304502210082bb528c57eb739edd87307fdc55f6f22dc33ef16aecdf3b42d8ad5c6b6ee7f202207cd93b27fb31301abfb1ef2f6dc3b96c0e059ecd4433933129e849126ac216d5
-    output htlc_success_tx 4: 0200000000010182fafbf148ac73a8a8cbbdf459408e1d368b4a2498bb0305a703cb800ae03b0e04000000000000000001d90d0000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500483045022100d388ded6a2cb9ebd6480342bd48ba28543b77bea5ea08c4b4b4a11751a13ced7022060aea2991e0f8c45a588ec0bf1bfc8ea939ee45e12edadba103d6fb49e2f76be0148304502210082bb528c57eb739edd87307fdc55f6f22dc33ef16aecdf3b42d8ad5c6b6ee7f202207cd93b27fb31301abfb1ef2f6dc3b96c0e059ecd4433933129e849126ac216d5012004040404040404040404040404040404040404040404040404040404040404046e21039390232673a9de88820d44ea910f364a332dc815cb0122bf5088d581dcbac8787c8201208763a91418bc1a114ccf9c052d3d23e28d3b0a9d1227434288527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae677502f801b175ac6800000000
+    remote_htlc_signature = 30450221009039b990b12951fcf180df6107b226d018caa8291f7ba753d22bf2b09b7d7f7b022060e9b05bcf6e2ab0ded7ae7aae179a4eccc7a3a39912da89a2468c292c549b59
+    # local_signature = 3045022100c9328a474ca222952a1cd25df57ea65b577cbf824752aed7fde4af1609b83b9202202888d5252a2d5949b288b0002f1019ba13c3c1a8e31ca5eb9ccab649bfebb789
+    output htlc_success_tx 0: 020000000001014a13eb73434fa97218577b183a4f2051b307dcee3712318b96eccd98be63c73a0000000000000000000121020000000000002200204adb4e2f00643db396dd120d4e7dc17625f5f2c11a40d857accc862d6b7dd80e0500473044022021413c112f8a8bb8343e5508acc11748ecc1dcf6a11b51534ce18a4af3ab252702200cda8cfaaaeb9f759ae551d3e67837fa8da6dd6396306ffeb98c3275f331546101483045022100c9328a474ca222952a1cd25df57ea65b577cbf824752aed7fde4af1609b83b9202202888d5252a2d5949b288b0002f1019ba13c3c1a8e31ca5eb9ccab649bfebb789012000000000000000000000000000000000000000000000000000000000000000006e210394854aa6eab5b2a8122cc726e9dded053a2184d88256816826d6231c068d4a5b7c8201208763a914b8bcb07f6344b42ab04250c86a6e8b75d3fdbbc688527c21030d417a46946384f88d5f3337267c5e579765875dc4daca813e21734b140639e752ae677502f401b175ac6800000000
```

Comment:
Now I have doubts about this one: shouldn't the output amount be 546 satoshis instead of 546, since the fee should be rounded down to 554 satoshis ? (it's also true with the current test vectors, I'm pretty sure we were not computing the output properly)